src/frontend/utils/chat/chat.ts

```diff
@@ -0,0 +1,31 @@
+import { converse } from 'utils/window';
```

Comment:
This is never used but at first I thought that was the main purpose of this PR.

Also why don't you use a lib to generate the id like [uuid](https://yarnpkg.com/package/uuid) ?

Reply:
I need to have control of what is generated for the `Anonymous-xxx...` username, what can be generated depends on what can converse allow as a chat's nickname. 
However, for the anonymous ID I assumed local storage and database can accept any type of string, so i will remove this part and use uuid instead, when I will implement this part.